components/AnimatedRegion.js

```diff
@@ -1,45 +1,50 @@
 /* eslint-disable */
+import {Animated} from 'react-native';
 
-class AnimatedRegion extends AnimatedWithChildren {
-  // latitude: AnimatedValue;
-  // longitude: AnimatedValue;
-  // latitudeDelta: AnimatedValue;
-  // longitudeDelta: AnimatedValue;
-  // _listeners: {[key: string]: {
+const AnimatedWithChildren = Object.getPrototypeOf(Animated.ValueXY);
+
+var _uniqueId = 1;
+
+export default class AnimatedMapRegion extends AnimatedWithChildren {
+  //latitude: AnimatedValue;
+  //longitude: AnimatedValue;
+  //latitudeDelta: AnimatedValue;
+  //longitudeDelta: AnimatedValue;
+  //_listeners: {[key: string]: {
   //  latitude: string,
   //  longitude: string,
   //  latitudeDelta: string;
   //  longitudeDelta: string,
-  // }};
+  //}};
 
   constructor(valueIn) {
     super();
-    const value = valueIn || { // probably want to come up with better defaults
+    var value = valueIn || { // probably want to come up with better defaults
       latitude: 0,
       longitude: 0,
       latitudeDelta: 0,
-      longitudeDelta: 0,
+      longitudeDelta: 0
     };
-    this.latitude = value.latitude instanceof Animated
+    this.latitude = value.latitude instanceof Animated.Value
```

Comment:
I've had a look but it doesn't seem possible to get `.interpolate, Animated.add, etc..` working without having to modify `AnimatedImplementation.js`. The crux is in the `maybeVectorAnim` function which explicitly checks for ValueXY class.
https://github.com/facebook/react-native/blob/5deb528695a8a47156a331b2a0182fabef0253d7/Libraries/Animated/src/AnimatedImplementation.js#L1831

 If we could replace that function with an extended one that support AnimatedRegion, then `.interpolate, Animated.multiply, etc..` should work with AnimatedRegion. It would have to be changed to this:

```js
var maybeVectorAnim = function(
  value: AnimatedValue | AnimatedValueXY | AnimatedRegion,
  config: Object,
  anim: (value: AnimatedValue, config: Object) => CompositeAnimation
): ?CompositeAnimation {
  if (value instanceof AnimatedValueXY) {
    var configX = {...config};
    var configY = {...config};
    for (var key in config) {
      var {x, y} = config[key];
      if (x !== undefined && y !== undefined) {
        configX[key] = x;
        configY[key] = y;
      }
    }
    var aX = anim((value: AnimatedValueXY).x, configX);
    var aY = anim((value: AnimatedValueXY).y, configY);
    // We use `stopTogether: false` here because otherwise tracking will break
    // because the second animation will get stopped before it can update.
    return parallel([aX, aY], {stopTogether: false});
  }
  else if (value instanceof AnimatedRegion) {
    var configLat = {...config};
    var configLng = {...config};
    var configLatDelta = {...config};
    var configLngDelta = {...config};
    for (var key in config) {
      var {lat, lng, latDelta, lngDelta} = config[key];
      if (lat !== undefined && lng !== undefined && latDelta !== undefined && lngDelta !== undefined) {
        configLat[key] = lat;
        configLng[key] = lng;
        configLatDelta[key] = latDelta;
        configLngDelta[key] = lngDelta;
      }
    }
    var aLat = anim((value: AnimatedRegion).latitude, configLat);
    var aLng = anim((value: AnimatedRegion).longitude, configLng);
    var aLatDelta = anim((value: AnimatedRegion).latitudeDelta, configLatDelta);
    var aLngDelta = anim((value: AnimatedRegion).longitudeDelta, configLngDelta);
    // We use `stopTogether: false` here because otherwise tracking will break
    // because the second animation will get stopped before it can update.
    return parallel([aLat, aLng, aLatDelta, aLngDelta], {stopTogether: false});
  }
  return null;
};
```